lib/OpenQA/JobTemplates.pm

```diff
@@ -21,12 +21,46 @@ use Exporter 'import';
 use Carp;
 use Try::Tiny;
 use JSON::Validator;
+use YAML::PP 0.020;
 
 our $VERSION   = '0.0.1';
 our @EXPORT_OK = qw(
-  &validate_data
+  &validate_data &load_yaml &dump_yaml
 );
 
+my $yp;
+
+sub _init_yaml_processor {
+    $yp ||= YAML::PP->new(
+        schema  => [qw/ + Merge /],
```

Comment:
What's this mean? Would you mind adding a comment, please? Or a link to the relevant documentation.

Reply:
I clarified what it means in comments.